weather_mv/README.md

```diff
@@ -274,6 +274,110 @@ weather-mv rg --uris "gs://your-bucket/*.nc" \
 For a full list of how to configure the Dataflow pipeline, please review
 [this table](https://cloud.google.com/dataflow/docs/reference/pipeline-options).
 
+### `weather-mv earthengine`
+
+```
+usage: weather-mv earthengine [-h] -i URIS --tiff_location TIFF_LOCATION --ee_asset EE_ASSET
+                           [--disable_grib_schema_normalization] [--use_personal_account]
+                           [--xarray_open_dataset_kwargs XARRAY_OPEN_DATASET_KWARGS] [--disable_in_memory_copy] [-s]
+```
+
+The `earthengine` subcommand ingests weather data into Earth Engine. In addition to the common options above, users may specify
+command-specific options:
+
+_Command options_:
+
+* `--tiff_location`: (required) Bucket location at which tiff files will be pushed.
+* `--ee_asset`: (required) The asset folder path in earth engine project where the tiff image files will be pushed.
+It should be in format: `projects/<project-id>/assets/<asset-folder>`. Make sure that <asset-folder> is there under <project-id>
+in earth engine assets
+i.e. projects/anthromet-prod/assets/ecmwf-data
+* `--disable_grib_schema_normalization`:  Restricts merging of grib datasets. Default: False
+* `-u, --use_personal_account`: To use personal account for earth engine authentication.
+* `--xarray_open_dataset_kwargs`: Keyword-args to pass into `xarray.open_dataset()` in the form of a JSON string.
+* `--disable_in_memory_copy`: Restrict in-memory copying of dataset. Default: False.
+* `-s, --skip-region-validation` : Skip validation of regions for data migration. Default: off.
+
+Invoke with `ee -h` or `earthengine --help` to see the full range of options.
+
+_Usage examples_:
+
+```bash
+weather-mv earthengine --uris "gs://your-bucket/*.grib" \
+           --tiff_location "gs://$BUCKET/tiffs" \  # Needed to store tiffs generated from *.grib
+           --ee_asset "projects/$PROJECT/assets/test_dir"
+```
+
+Using the subcommand alias `ee`:
+
+```bash
+weather-mv ee --uris "gs://your-bucket/*.grib" \
+           --tiff_location "gs://$BUCKET/tiffs" \  # Needed to store tiffs generated from *.grib
+           --ee_asset "projects/$PROJECT/assets/test_dir"
+```
+
+Preview ingestion with a dry run:
+
+```bash
+weather-mv ee --uris "gs://your-bucket/*.grib" \
+           --tiff_location "gs://$BUCKET/tiffs" \  # Needed to store tiffs generated from *.grib
+           --ee_asset "projects/$PROJECT/assets/test_dir"
+           --dry-run
+```
+
+Restrict in-memory copying of dataset:
+
+```bash
+weather-mv ee --uris "gs://your-bucket/*.grib" \
+           --tiff_location "gs://$BUCKET/tiffs" \  # Needed to store tiffs generated from *.grib
+           --ee_asset "projects/$PROJECT/assets/test_dir"
+           --disable_in_memory_copy
+```
+
+Authenticate earth engine using personal account:
```

Comment:
Correct. On the browser window, user will need to select the account and "allow" earth engine authenticator.

Reply:
Only on the local runner. For this user has to specify the -u/--use_personal_account flag. But yes, if provided the flag and running on Dataflow runner, it would not work.
I should validate this, right? (Dataflow and -u should not be together.)